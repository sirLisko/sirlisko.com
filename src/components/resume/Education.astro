<article class="section education">
  <h1>Education</h1>
  <p>
    Degree in Computer Science @ Universit√† degli studi dell'Insubria &amp;
    Universidad de Salamanca.
  </p>
  <p>
    Thesis on Assistive Technology -{" "}
    <a href="https://assistivetechnology.it">https://assistivetechnology.it</a>
  </p>
</article>
