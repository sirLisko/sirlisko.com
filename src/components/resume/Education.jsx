import React from "react";

const Education = () => (
  <article className="section education">
    <h1>Education</h1>
    <p>
      Degree in Computer Science @ Università degli studi dell’Insubria &amp;
      Universidad de Salamanca.
    </p>
    <p>
      Thesis on Assistive Technology -{" "}
      <a href="http://assistivetechnology.it" data-beacon>
        http://assistivetechnology.it
      </a>{" "}
      (sorry link in Italian).
    </p>
  </article>
);

export default Education;
