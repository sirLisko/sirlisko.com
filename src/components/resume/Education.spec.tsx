import React from "react";
import { render } from "@testing-library/react";

import Education from "./Education";

describe("Education Component", () => {
  it("should render properly", () => {
    const { asFragment } = render(<Education />);
    expect(asFragment()).toMatchSnapshot();
  });
});
