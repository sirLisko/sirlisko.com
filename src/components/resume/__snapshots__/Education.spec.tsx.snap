// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Education Component should render properly 1`] = `
<DocumentFragment>
  <article
    class="section education"
  >
    <h1>
      Education
    </h1>
    <p>
      Degree in Computer Science @ Università degli studi dell’Insubria & Universidad de Salamanca.
    </p>
    <p>
      Thesis on Assistive Technology - 
      <a
        href="http://assistivetechnology.it"
      >
        http://assistivetechnology.it
      </a>
       (sorry link in Italian).
    </p>
  </article>
</DocumentFragment>
`;
